provider/resource.go

```diff
@@ -590,6 +590,8 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_custom_provider":                    {Tok: azureResource(azureCore, "CustomProvider")},
 			"azurerm_resource_provider_registration":     {Tok: azureResource(azureCore, "ResourceProviderRegistration")},
 			"azurerm_subscription":                       {Tok: azureResource(azureCore, "Subscription")},
+			"azurerm_tenant_template_deployment":         {Tok: azureResource(azureCore, "TenantTemplateDeployment")},
+			"azurerm_template_spec_version":              {Tok: azureResource(azureCore, "TemplateSpec")},
```

Comment:
Should it not be `TemplateSpecVersion`?

Reply:
great catch! This should actually have been a datasource as well :/